newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -105,6 +105,12 @@ common: &default_settings
       # Default is 10000. Setting to 0 will disable.
       #max_samples_stored: 10000
 
+      # Whether the log events should include MDC from loggers with support for that.
+      include_context_data:
+
+        # When true, application logs will contain context data.
+        enabled: false
```

Comment:
I wonder if we should also put the `include`/`exclude` options in the default yaml but keep them commented out like we do elsewhere?

```
      # A comma separated list of attribute keys whose values should be sent to New Relic.
      #include:

      # A comma separated list of attribute keys whose values should not be sent to New Relic.
      #exclude:
```